source/nodejs/adaptivecards-designer/src/containers/federated-search/federated-search-container.ts

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as Adaptive from "adaptivecards";
+import { HostContainer } from "../host-container";
+import * as hostConfig from "../../hostConfigs/federated-search.json";
+
+export class FederatedSearchContainer extends HostContainer {
+  constructor(name: string, styleSheet: string) {
+    super(name, styleSheet);
+
+    this.actionsRegistry.unregister("Action.Submit");
+    this.elementsRegistry.unregister("Input.Text");
+    this.elementsRegistry.unregister("Input.Date");
+    this.elementsRegistry.unregister("Input.Time");
+    this.elementsRegistry.unregister("Input.Toggle");
+    this.elementsRegistry.unregister("Input.ChoiceSet");
+    this.elementsRegistry.unregister("Input.Number");
+    this.actionsRegistry.register("Action.Http", Adaptive.HttpAction);
+    this.cardHost.classList.add("fedsearch-card");
+  }
+
+  public renderTo(hostElement: HTMLElement) {
+    hostElement.classList.add("outlook-frame");
+    hostElement.appendChild(this.cardHost);
+  }
+
+  public initialize() {
+    super.initialize();
+
+    Adaptive.GlobalSettings.useMarkdownInRadioButtonAndCheckbox = false;
+  }
+
+  private parsePadding(source: any): Adaptive.PaddingDefinition {
```

Comment:
This seems to have been copied straight from the Outlook host container. I doubt Federated Search cards support explicit padding (and if they do they probably shouldn't). This method likely should be removed.